astro/src/content/articles/tokens/revoking-jwts.md

```diff
@@ -29,6 +29,8 @@ The ToDo Backend in the diagram can use the JWT and the public key to verify the
 
```

Comment:
We should really consider updating the diagram above (`jwt-revoke_350.png`) if this is a high traffic page.

Reply:
Yes! https://linear.app/fusionauth/issue/DES-560/article-or-revoking-jwts-or-articles